src/main/java/ch/njol/skript/expressions/ExprItemThrower.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Thrower")
+@Description("The player that thrown the dropped item.")
+@Examples({
+	"broadcast the item thrower of all dropped items"
+})
+@Since("INSERT VERSION")
+public class ExprItemThrower extends SimplePropertyExpression<Entity, OfflinePlayer> {
+
+	static {
+		registerDefault(ExprItemThrower.class, OfflinePlayer.class, "[item] thrower", "entities");
+	}
+
+	@Override
+	public @Nullable OfflinePlayer convert(Entity entity) {
+		if (!(entity instanceof Item item))
+			return null;
+		UUID uuid = item.getThrower();
+		if (uuid == null)
+			return null;
+		return Bukkit.getOfflinePlayer(uuid);
+	}
+
+	@Override
+	public Class<?> @Nullable [] acceptChange(ChangeMode mode) {
+		if (mode == ChangeMode.SET || mode == ChangeMode.DELETE)
+			return CollectionUtils.array(OfflinePlayer.class, String.class, UUID.class);
+		return null;
```

Comment:
Additional note, this should include Entity not only OfflinePlayer (will still need offlineplayer) the same goes for the owner expression